### DISCUSSION TITLE
Categories are not showing

### DISCUSSION THREAD

**Discussion by adithyaprakashv** [learndevopsonline/learndevopsonline#1258] [Q&A] [answered]
I can't able to see the categories, I followed the sessions steps everything, I cross checked with documents as well. But I don't know where I am doing mistake I can't able to see the categories. Can you Please help me with this
frontend IP address: 44.210.112.235
Mongodb IP address: 3.88.181.136
Catalouge IP address: 3.88.186.147

![Screenshot_7](https://github.com/learndevopsonline/learndevopsonline/assets/137979935/a03a5920-4f08-46ae-94f6-949db0d2334e)
![Screenshot_8](https://github.com/learndevopsonline/learndevopsonline/assets/137979935/b6a0d026-6ec5-4575-b650-597f0e25ce12)

**Comment by maheshbusireddy** [comment, marked as answer]
@adithyaprakashv you need to update the robohsop.conf file in server. you have pull the code from git hub properly.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/99891108/5c7db9bb-0932-48ba-ad06-8c1c1941b464)
